doc/introduction/interfaces/jax.rst

```diff
@@ -0,0 +1,137 @@
+.. _jax_interf:
+
+JAX interface
+=================
+
+In order to use PennyLane in combination with JAX, we have to generate JAX-compatible

```

Comment:
Since 80% of our users may not come from ML and have only vaguely heard of JAX, would it be good to add a sentence of why they should use it? I.e., "Born out of the autograd package, JAX is optimised for speed and allows  XXXX"

Reply:
Oh, and a link to the JAX docs as well!

Reply:
Added.